program-runtime/src/loaded_programs.rs

```diff
@@ -1004,13 +977,14 @@ impl<FG: ForkGraph> LoadedPrograms<FG> {
                         _ => None,
                     })
             })
-            .sorted_by_cached_key(|(_id, program)| program.tx_usage_counter.load(Ordering::Relaxed))
             .collect()
     }
 
     /// Unloads programs which were used infrequently
     pub fn sort_and_unload(&mut self, shrink_to: PercentageInteger) {
```

Comment:
Is `sort_and_unload` still used somewhere?

Reply:
No, left it in the code for time being, in case we need to go back to it. Let's get the new code to testnet and if it works well we can remove it.